.ci/yamllint.yml

```diff
@@ -9,3 +9,5 @@ rules:
   truthy: disable
   # 80 chars should be enough, but don't fail if a line is longer
   line-length: disable
+  comments:
+    min-spaces-from-content: 1
```

Comment:
why do we need this change?

Reply:
To fix sdk yamllint issue https://github.com/networkservicemesh/sdk/actions/runs/10899858521/job/30246211877?pr=1664 `too few spaces before comment  (comments)`. We have discussed, that this linter config update is ok.

Reply:
What exactly line(change) causes  the problem in https://github.com/networkservicemesh/sdk/pull/1664?

Reply:
./.golangci.yml
  60:14     warning  too few spaces before comment  (comments)
  61:14     warning  too few spaces before comment  (comments)
  62:14     warning  too few spaces before comment  (comments)

      - G301 # Fixable
      - G302 # Fixable
      - G306 # Fixable